dts/bindings/sensor/we,wsen-hids-2525020210002.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2023 Würth Elektronik eiSos GmbH & Co. KG
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Würth Elektronik WSEN-HIDS-2525020210002 humidity sensor
+
+compatible: "we,wsen-hids-2525020210002"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+
+  precision:
+    type: string
+    default: "Medium"
```

Comment:
The default value needs to be justified in the description. For example, is it a hardware register reset value?

Reply:
default value explanation has been added.